projects/vdk-plugins/vdk-jupyter/getting-started.ipynb

```diff
@@ -0,0 +1,292 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d350971c",
+   "metadata": {},
+   "source": [
+    "# Welcome to Versatile Data Kit \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "575951ae",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "Welcome to our introductory guide. \n",
+    "Here, we'll walk you through the process of creating, developing, and deploying a Data job using the Jupyter UI. \n",
+    "Please take the time to carefully go through this guide to ensure a smooth and effective understanding on using VDK."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bdeac937-04bb-4e46-89b1-e946406e5cd3",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "<h2>What exactly is a Data Job?</h2>\n",
+    "    <p>Data job is a data processing unit that allows data engineers to implement automated pull ingestion (E in ELT) or batch data transformation into Data Warehouse (T in ELT). At the core of it, it is a directory with different scripts and inside of it.</p>\n",
+    "\n",
+    "<h4>Data Job directory can contain any files, however, there are some files that are treated in a specific way.</h4>\n",
+    "\n",
+    " \n",
+    "<details>\n",
+    "        <summary><strong style=\"color: Tomato;\">Click to see them!</strong></summary>\n",
+    "        <ul>\n",
+    "            <li><strong style=\"color: blue;\">SQL files (.sql)</strong> - called SQL steps - are directly executed as queries against a database.</li>\n",
+    "            <li><strong style=\"color: green;\">Notebook files (.ipynb)</strong> - can contain labelled VDK cells (called notebook steps) which are executed as steps in a VDK Data Job;</li>\n",
+    "            <li><strong style=\"color: red;\">Python files (.py)</strong> - called Python steps - are python scripts that define a run function that takes as an argument the job_input object.</li>\n",
+    "            <li><strong style=\"color: purple;\">config.ini</strong> is needed in order to schedule a job.</li>\n",
+    "            <li><strong style=\"color: brown;\">requirements.txt</strong> is needed when your Python or Notebook steps use external python libraries.</li>\n",
+    "        </ul>\n",
+    "        <img src=\"./images/data-job-directory.png\" width=\"1000\" length=\"546\" alt=\"Data Job Directory\">\n",
+    "</details>\n",
+    "\n",
+    "\n",
+    "\n",
+    "   <p>VDK supports having many Python, SQL and Notebook steps in a single Data Job. These steps are carried out in an ascending alphabetical sequence based on the file names, and Notebook steps are run from the beginning to the end of the file. For an effective execution order while retaining informative file names, it's beneficial to prefix your file names with numbers.</p>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eb565197",
+   "metadata": {},
+   "source": [
+    "<h2>Step 0: Explore VDK's Functionalities</h2>\n",
```

Comment:
We can skip that. People would not be using CLI so !vdk --help would be just confusing since most of functionality is duplicated through the UI or the notebook